lib/couchbase-orm/utilities/query_helper.rb

```diff
@@ -132,8 +132,11 @@ def serialize_value(key, value_before_type_cast)
             end
 
             def quote(value)
-                if [String, DateTime, Date, Time].any? { |clazz| value.is_a?(clazz) }
```

Comment:
Active record datetime columns are mapped to `ActiveSupport::TimeWithZone` ruby class. Which is a `Time` (And not a `DateTime` )